buildozer/target.py

```diff
@@ -102,7 +102,7 @@ def cmd_update(self, *args):
     def cmd_debug(self, *args):
         self.buildozer.prepare_for_build()
         self.build_mode = 'debug'
-        self.artifact_format = 'apk'
+        self.artifact_format = self.buildozer.config.getdefault('app', 'android.debug_artifact', 'apk')
```

Comment:
Can you also add `android.debug_artifact` to `default.spec` ? (Near to `android.release_artifact` would be nice)

Reply:
Done